pkg/kubelet/status/status_manager.go

```diff
@@ -140,6 +140,12 @@ type Manager interface {
 	// State returns a read-only interface to the internal status manager state.
 	State() state.Reader
```

Comment:
looks like we have no external callers of State() now... given we're unsure whether it will be nil or not, can we drop this accessor from the interface to keep all nil-handling local?

Reply:
Yes, this can go.